docs/topics/proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc

```diff
@@ -86,7 +87,7 @@ The number in the middle will increase with each new build.
 The letters at the end are generated when the pod is created.
 
 
-. Once your booster is deployed and started, determine its route.
+. Once your example is deployed and started, determine its route.
```

Comment:
Once your example is deployed --> After your example application is deployed

Reply:
Updated similar modules as well.